dataprofiler/profilers/json_decoder.py

```diff
@@ -74,6 +75,31 @@ def get_option_class(class_name: str) -> type[BaseOption]:
     return options_class
 
 
+def get_profiler_class(class_name: str) -> type[BaseProfiler]:
+    """
+    Use name of class to return default-constructed version of that class.
+
+    Raises ValueError if class_name is not name of a subclass of
+        BaseProfiler.
+
+    :param class_name: name of BaseProfiler subclass retrieved by
+        calling type(instance).__name__
+    :type class_name: str representing name of class
+
+    :raises: NotImplementedError()
+    :raises: ValueError()
+
+    :return: subclass of BaseProfiler object
+    """
+    if class_name == "UnstructuredProfiler":
```

Comment:
changes

Reply:
I'm not sure we need this since below:
`raise ValueError(f"Invalid profiler class {class_name} " f"failed to load.")`

Reply:
yeah I think this is superfluous